coordinator/src/main/scala/filodb/coordinator/v2/FiloDbClusterDiscovery.scala

```diff
@@ -99,7 +107,7 @@ class FiloDbClusterDiscovery(settings: FilodbSettings,
     val empty = CurrentShardSnapshot(ref, new ShardMapper(numShards))
     def fut = (nca ? GetShardMapScatter(ref)) (t).asInstanceOf[Future[CurrentShardSnapshot]]
     Observable.fromFuture(fut).onErrorHandle { e =>
-      logger.error(s"Saw exception on askShardSnapshot: $e")
```

Comment:
This will log even if clustering V1 is enabled?

Reply:
Could you put the exception as the second argument to `logger.error`? I see it wasn't there before, but that will make stack traces more conventional.

Reply:
We use this class only for V2. So it won't be logged.

Surely, updating the exception log